tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -227,6 +227,34 @@ def _check_trt_version_compatibility():
         "This is supported because TensorRT " +
         " minor/patch upgrades are backward compatible")
 
+def _disable_non_trt_optimizers_in_rewriter_config(rewriter_config):
```

Comment:
We should try to avoid putting this routine here, instead put it in something like compiler/tensorrt/trt_utils.py.
I am asking around if there is a better way to turn off all the optimizations in a rewriter config and will get back to you once I hear back some answers.

Reply:
can we put this routine to a new file such as compiler/tensorrt/trt_utils.py?

Reply:
I'm creating that utils.py file then. Will send the update tomorrow.

I searched the whole TF codebase and I haven't seen such a "functionality". I would be very surprised if it does exist.